addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -1430,9 +1430,13 @@ export class OdooEditor extends EventTarget {
         const block = closestBlock(sel.anchorNode);
         for (const [style, tag, isList] of [
             ['paragraph', 'P', false],
+            ['pre', 'PRE', false],
             ['heading1', 'H1', false],
             ['heading2', 'H2', false],
             ['heading3', 'H3', false],
+            ['heading4', 'H4', false],
```

Comment:
Not really because this loop is only to set a button active in case the range is at a specific tag.
But it turns out that there is no such button in 14.4 so this does not change anything.

Actually I don't think this commit is useful anymore as I changed the logic that shows under which tag the range is, through the "selection widget":
![](https://i.imgur.com/nSAWsE9.png)
When stefano originally made the commit, the logic was to have button inside the "selection widget" with only one being active.

But we can keep this commit nonetheless as we could theoretically have button in the future.

But if you want I remove the commit.

Reply:
I took 10 seconds to actually test in 14.4 and saw that h1 -> h3 buttons inside the select were marked as active when selecting the related tag but h4 -> h6 were never marked as active. I applied Stefano's commit and it solved the problem. While I was at it, I made the PR and r+'ed: https://github.com/odoo/odoo/pull/74906 :shrug:

Reply:
Thank's. I didn't tested it but I should have. :+1: